R/resample.R

```diff
@@ -147,6 +147,9 @@ resample_workflow <- function(workflow, resamples, metrics, control) {
   check_rset(resamples)
   check_workflow(workflow)
   metrics <- check_metrics(metrics, workflow)
+  has_formula <- has_preprocessor_formula(workflow)
```

Comment:
Why is this here?

Reply:
Most likely vestigial code. It's not used so I'll remove it.